vic/vic_run/include/vic_log.h

```diff
@@ -54,7 +54,7 @@
 // | INFO      | < 20             |
 // | DEBUG     | < 10             |
 #ifndef LOG_LVL
-#define LOG_LVL 25
+#define LOG_LVL 5
```

Comment:
reminder to change back to 25 after debugging / changes to `ccsm_utils/Machines/config_compilers.xml` are committed.